CHANGELOG.md

```diff
@@ -24,7 +24,8 @@ folder structure is present irrespective of using InnerEye as a submodule or not
 environment will be contained in the model.
 
 ### Removed
-- Removed blobxfer completely. AzureML Data-stores for reading datasets make the following configs obsolete: 'datasets_storage_account' and 'datasets_storage_account_key' and they are not longer supported. 
+- Removed blobxfer completely. When downloading a dataset from Azure, we now use AzureML dataset downloading tools.
+Please remove the following fields from your settings.yml file: 'datasets_storage_account' and 'datasets_container'. 
```

Comment:
datasets_storage_account_key was removed from the comments?

Reply:
There never was such a field.